notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -356,3 +356,46 @@ demon [Bill and Ted met a pair of rather friendly demons during their brief stay
+
+:: Title
+2067
+
+:: Description
+In the year 2067, Earth has been devastated by climate change, and it is up to
+one man to travel 407 years into the future to set things straight.
+
+:: Date
+2020-10-02
+
+:: References
+https://en.wikipedia.org/wiki/2067_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+what if the fate of the world was in my hands [It was up to Ethan to travel 407 years into the future to find a cure for a strange sickness that was going to wipe out humanity.]
+global warming [The film is set 2067 when Earth has been devastated by climate change. All plant life on Earth had gone extinct, and civilization had collapsed around the globe with only one city in Australia still holding out.]
+time machine [A main novelty of the film is the large, nuclear powered, stationary time machine that Ethan used to travel 407 years into the future.]
+husband and wife [Ethan was caring for his ill wife, Xanthe, and missed her deeply after he traveled into the future.]
+coping with a loved one being gravely ill [Ethan's beloved wife, Xanthe, was dying from a strange affliction that was a result of her breathing tainted oxygen.]
+destiny [Ethan was convinced that it was his fate (a word that is uttered five times in the film) that Jude would shoot him dead, and in so doing put an end to Ethan's effort to save humanity. Toward the end, however, Ethan changed his tune an resolved to change his own fate, and the fate of humanity by extension.]
+father and son [Central to the story was Ethan's dealing with his daddy issues.]
+
+:: Minor Themes
+coping with being in a heat wave [A reporter spoke of a record-breaking heat wave.]
+hologram [People communicated via holographic recordings.]
+suicide [A guilt-ridden Jude took his own life. It was said that Ethan's father had committed suicide when Ethan was only eight years old. Although it was later revealed that the Chronicorp CTO had shot him dead.]
+love vs. the public good [Should Ethan stay in the present to care for his gravely ill wife or go on a possibly one way trip into the future in a last ditched effort to save humanity?]
+wilderness survival [Ethan rubbed two sticks together to make a fire to survive the night in the rainforest.]
+personal identity [Ethan was puzzled about if his future skeleton was him.] {48:30}
+cold fusion [The handheld computer Archie detected a corrupted cold fusion power feed in the time machine's nuclear core.]
+mother and son [A young boy watched his mother die of a gunshot wound to the neck.]
+remorse [a guilt-ridden Ethan spoke of how he'd left his wife to die. Richard lamented that he hadn't been there for his son Ethan.]
+parental love [Richard pointedly proclaimed his love to his son Ethan.]
+choosing who lives and who dies [Regina Jackson selected the "chosen few" who would be sent 407 years into the future to reboot civilization.]
+time portal [The prototype time machine worked by generating a portal of some sort that one could pass through.]
```

Comment:
I thought these things ought to be more or less permanent fixtures to qualify - our Wikipedia reference concurs with me though I see our own definition text could use a little TLC.

Reply:
At any rate I really think we should find a way not to use it when it's merely a time machine that briefly opens up a portal like opening. Usually that's just as convenient visual effect and doesn't make the machine different from any other time machine.

Reply:
Very good. I took it out. I was annoyed that I felt compelled by our definition to use it in the first place. I'll submit a new pull request with a revised version of the theme.